src/App.tsx

```diff
@@ -17,18 +80,36 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                status={status}
+                query={query}
```

Comment:
Also you can take this values from store. It discussable that better pass it as prop or use from store, but I relates on redux recommendations:
 https://redux.js.org/style-guide/#connect-more-components-to-read-data-from-the-store